src/e2e/puppeteer/__tests__/ContextView.ts

```diff
@@ -33,6 +34,8 @@ it('load buffered ancestors of contexts when context view is activated', async (
   await sleep(100)
 
   await clickThought('m')
+  await scrollIntoView('[data-testid="toolbar-icon"][aria-label="Note"]')
```

Comment:
Scrolling to the Note button is no better than scrolling to Settings button. You are still tightly coupling the Context View test to an unrelated button. What happens if the buttons change position in the Toolbar? Then this test will break. Instead you need to:

1. Scroll to the Context View button plus a few pixels.
2. Scroll to the "next" button after the Context View regardless of what that button is (using DOM traversal).